tcms/testcases/templates/testcases/mutable.html

```diff
@@ -40,7 +40,7 @@
                 <div class="col-md-3 col-lg-3 {% if form.product.errors %}has-error{% endif %}">
                     <select id="id_product" name="product" class="form-control selectpicker">
                         {% for product in form.product.field.queryset %}
-                            <option value="{{ product.pk }}" {% if product.pk|escape == form.product.value|escape %}selected{% endif %}>
+                            <option value="{{ product.pk }}" {% if product.pk|escape ==  object.category.product_id|escape %}selected{% endif %}>
```

Comment:
Why change here ?

Reply:
Previously the old form class was initiated with 'product': test_case.category.product_id. Now I use new form class, that`s why I must get the value from the model(object).

Reply:
There is still a product field in the new form b/c user can change test cases between products. So this change is not necessary.